### ISSUE TITLE
Something up with the CI?

### ISSUE THREAD

**Issue by gareth-ellis** [nodejs/build#822]
It seems there's something up with the CI; new jobs are sat waiting for hosts that are idle, and currently running jobs seem to have been running for over 10 hours....

Can someone take a look?

**Comment by gibfahn** [comment]
Yep, not sure what's going on here:

###Waiting for executor:

![image](https://user-images.githubusercontent.com/15943089/28964139-7c05f38e-7903-11e7-8cce-364a930de3f1.png)

###Idle executor:

![image](https://user-images.githubusercontent.com/15943089/28964148-81687de2-7903-11e7-846f-079f18900b06.png)

**Comment by gareth-ellis** [comment]
Just to point out too, it doesnt seem to just be benchmarking stuff, there's a lot of stuff in the queue waiting for machines that are idle:
![image](https://user-images.githubusercontent.com/14981026/28964318-93530820-790c-11e7-838e-8e69068f8270.png)

**Comment by refack** [comment]
Is this something @nodejs/jenkins-admins can resolve?

**Comment by refack** [comment]
AFAICT the first job in the queue is a `libuv` job so maybe cancelling it will kick start Jenkins 🤷‍♂️ 
/cc @libuv @saghul

**Comment by rvagg** [comment]
I cancelled most things in the queue and it seems to have kicked it into gear again. I'm not entirely sure what them problem is but it seemed to be stuck on a windows restart job, one of the expected servers wasn't available for that one to complete. Also a libuv job was having trouble. I think that one may be due to the armv7-ubuntu1404 machines being unavailable. I've had to take them out completely because they all have problems (power supply problems I'm guessing). So I've removed armv7-ubuntu1404 and the Pi's from libuv test commit for now. 

:zzz:
